benchmark.py

```diff
@@ -271,7 +272,7 @@ def store_result(key, doc):
     default_doc = create_doc(args)
     default_doc['meta']['versions'] = {
         'python': '.'.join(map(str, sys.version_info)),
-        'signac': signac.__version__,
+        'signac': __version__,
```

Comment:
The changes to this module should not be necessary.

Reply:
The __version__ variable is now located at signac.version.__version__ not signac.__version__, so I think this change is necessary.